src/resources/projects/website/templates/sidebar.ejs

```diff
@@ -92,4 +92,4 @@
   <% } %>
 
 </nav>
-<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" ></div>
```

Comment:
I wonder if the lack of a newline at the end of this file was intentional, and that perhaps we don't want to change that.

Reply:
No, and I don’t know why/how it was removed as my settings basically is to always have a new line.

Reply:
In fact, there was no changes but GitHub does not like when there is no new line.

- https://github.com/quarto-dev/quarto-cli/blob/9724c3dd403d4e723bcb0a7e7bef6e49d23a353a/src/resources/projects/website/templates/sidebar.ejs